### PR TITLE
Apply remove unused imports on the rest of RenameClassRector test fixtures

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc [hunk 1]
```diff
@@ -22,7 +22,7 @@ namespace MyNewNamespace;
 class MyNewClass
 {
     /**
-     * @return \MyNewNamespace\MyNewClass
+     * @return MyClass
```
Comment (samsonasik) [inline rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc]: this should be kept as `MyClass` changed to `MyNewClass`

Comment (TomasVotruba) [reply rules-tests/Renaming/Rector/Name/RenameClassRector/Fixture/rename_class.php.inc]: Not sure why this one keeps changing :/ It has somethign to do with cache, possibly similar method info.

**Comment by TomasVotruba** [comment]
@samsonasik Could you review now? Just to be sure :)

**Review by samsonasik — APPROVED** [review]
as it is optional feature, I am ok with it 👍 , next fixture possibly needed for this use case for use as namespace:

```php
use A\B;

new B\C();
```

or grouped use:

```php
use Rector\Core\Tests\Issues\AutoImportGroupUse\Source\{ SomeClass };

new SomeClass();
```

above need to be kept.

**Comment by TomasVotruba** [comment]
Thanks :+1: 

Yeah, I'm testing is super slowly, so we have it covered for sure.

**Comment by samsonasik** [comment]
@TomasVotruba I created failing fixture PR for namespaced use https://github.com/rectorphp/rector-src/pull/3365

**Pull request merged**